src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -180,13 +182,52 @@ def composeurl2images(
     return [(composepath + qcow2[0].path) for qcow2 in candidates]
 
 
+def centoshtml2image(url, desiredarch):
+    """Find the latest image url for the CentOS Stream HTML image list."""
+    # we will need to join it with a relative path component
+    if url.endswith("/"):
+        path = url
+    else:
+        path = url + "/"
+    if path.find("/centos/9-stream/") > -1:
+        centosver = 9
+    elif path.find("/centos/8-stream/") > -1:
+        centosver = 8
```

Comment:
If `centos/9-stream` exists, we cannot choose the 8-stream image over 9 as long as we use centoshtml?

Reply:
`"centoshtml": "https://cloud.centos.org/centos/9-stream/x86_64/images"`

Reply:
thanks! i misread the version and the rest... :p